src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    protected String color;

```

Comment:
According to the Java style guide, the access modifier for class fields should be specified explicitly. In this case, it seems like 'color' and 'number' can be private as they are accessed via getters and setters.